lib/resources/example_json/r4_examples_document_reference.rb

```diff
@@ -140,148 +154,162 @@ module Resources
         }
       ],
       'entry': [
         {
-          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference/197411797',
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference/21961261',
           'resource': R4_DOCUMENT_REFERENCE
         },
         {
-          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference/197411793',
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/DocumentReference/21674656',
           'resource': {
             'resourceType': 'DocumentReference',
-            'id': '197411793',
+            'id': '21674656',
             'meta': {
               'versionId': '1',
-              'lastUpdated': '2020-12-10T07:19:56.000Z'
+              'lastUpdated': '2021-03-05T16:48:42.000Z'
             },
             'text': {
               'status': 'generated',
-              'div': '<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><b>Document Reference</b></p>'\
-                     '<p><b>Patient Name</b>: PETERS, TIM A</p><p><b>Document Type</b>: Progress Note Generic</p>'\
-                     '<p><b>Document Title</b>: Auto TC Summary</p><p><b>Service End Date</b>'\
-                     ': Dec 10, 2020  7:19 A.M. UTC</p><p><b>Document Status</b>: Final</p><p>'\
-                     '<b>Verifying Provider</b>: Portal, Portal</p></div>'
+              'div': '<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><b>Document Reference</b></p><p><b>Patient Name'\
+                     '</b>: Parnell, Stephen M</p><p><b>Document Type</b>: Abdominal Ultrasound</p><p><b>'\
+                     'Document Category</b>: Unknown</p><p><b>Document Title</b>: test images</p><p><b>'\
+                     'Service End Date</b>: Mar  5, 2021 10:47 A.M. CST</p><p><b>Document Status</b>: Preliminary</p>'\
+                     '</div>'
             },
+            'identifier': [
+              {
+                'system': 'https://fhir.cerner.com/ceuuid',
+                'value': 'CEfda49233-ccfa-4ed4-afbc-9f5082c2bf0c-21674656-2021030510484300'
+              }
+            ],
             'status': 'current',
-            'docStatus': 'final',
+            'docStatus': 'preliminary',
             'type': {
               'coding': [
                 {
                   'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/72',
-                  'code': '2820555',
-                  'display': 'Progress Note Generic',
+                  'code': '2820643',
+                  'display': 'Abdominal Ultrasound',
                   'userSelected': true
                 },
                 {
                   'system': 'http://loinc.org',
-                  'code': '11506-3',
-                  'userSelected': false
+                  '_code': {
+                    'extension': [
```

Comment:
Is this correct? Usually, a DAR wouldn't have a system?

Reply:
Yes, a LOINC coding containing a DAR had to be added for DocRef to satisfy US Core requirements

Reply:
This isn't correct.  All LOINC values whose SCALE is DOC in the LOINC database and the HL7 v3 Code System NullFlavor concept 'unknown'.
http://build.fhir.org/ig/HL7/US-Core/ValueSet-us-core-documentreference-type.html

Reply:
So because of the NullFlavor concept 'unknown' is why a DAR is necessary here (if I am understanding correctly)? Or does a change need to be made?

Reply:
What jira are you working on?  What has a NullFlavor?  The extension doesn't look correct and you can ignore my earlier comment.  I thought you were trying to add a DAR when there is no coding with a LOINC system.